keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -338,3 +339,170 @@ parameters: string:string [8]
 1. `secretRef`: (Optional) The name and namespace of an existing secret to be copied to the `Bucket`'s namespace for static provisioning.  Requires that `bucketIdentifier` be defined. Used for brownfield and static cases.
 1. `parameters`: (Optional) Object store specific key-value pairs passed to the driver.
 
+### RPC
+
+```
+syntax = "proto3";
+
+package cosi.v1;
+
+import "google/protobuf/descriptor.proto";
+
+option go_package = "github.com/container-object-store-interface/go-cosi";
+
+extend google.protobuf.MessageOptions {
+    bool cosi_secret = 1000;
```

Comment:
is there any concern about another app (process?) using the same extension number?

Reply:
no, unless they're importing our .proto files.  The extensions are scoped to the generated code.